src/renderer.ts

```diff
@@ -313,6 +313,23 @@ const richLinkWidth = '8.75rem';
 const richLinkStyles = css`
     background: ${neutral[97]};
     padding: ${basePx(1)};
+    border-top: solid 1px ${neutral[60]};
+    ${textSans.medium()}
+
+    p:last-child {
+        margin-bottom: 0;
+        padding-bottom: 0;
+    }
+
+    p::before {
+        ${icons}
+        content: '\\e005';
+        border-radius: 100%;
+        border: solid 1px ${neutral[7]};
+        padding: 4px 5px 5px 5px;
+        font-size: 12px;
```

Comment:
We might want these in `rem` to avoid scaling issues?

Reply:
I'm a bit worried about the circle looking less round at some sizes, I think we can definitely use rem sizing for the svg though https://github.com/guardian/source/issues/371